libpod/container_internal.go

```diff
@@ -633,11 +644,56 @@ func (c *Container) refresh() error {
 		return err
 	}
 
-	if rootless.IsRootless() {
+	return nil
+}
+
+// Try and remove IP address allocations. Presently IPv4 only.
+// Should be safe as rootless because NetworkStatus should only be populated if
+// CNI is running.
+func (c *Container) removeIPv4Allocations() error {
+	// Using a hardcoded path here is VERY gross, but the lack of a good
+	// interface into CNI means there's no non-hacky way to do this.
+	const cniNetworksDir = "/var/lib/cni/networks/"
```

Comment:
On the fence, should there be a function like getCNINetworksDir that returns this string when on a linux distro, and then unknown or something for windows/others?  Or maybe define this in containers.conf?

Reply:
Yes this should be handled in container_internal_linux.go.

Reply:
Fixed